.gitbook/3-protocol-devs/1-dep-specs/6-incentives.md

```diff
@@ -0,0 +1,66 @@
+# Incentives
+
+- Status: Draft
+- Created: 2023-01-06
+- Modified: 2023-01-06
+- Authors
+    - Youngjoon Lee <yjlee@medibloc.org>
+    - Gyuguen Jang <gyuguen.jang@medibloc.org>
+    - Hansol Lee <hansol@medibloc.org>
+    - Myongsik Gong <myongsik_gong@medibloc.org>
+    - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+When data providers [submit consent to provide their data](./3-data-provider-consent.md), the budget deposited by the data consumer is distributed to the provider and the oracle that verified the data.
+
+### Motivation
+
+This is for transparent distribution for the cost of providing data and the cost of verifying data.
+
+### Definitions
+
+`Data Provider`, `Data Consumer` and `Oracle` are defined in [User Flow](./1-user-flow.md)
+
+## Technical Specification
+
+### Reward Distribution
+
+When [creating a data deal](./2-data-deal.md#create-data-deal), data consumers specify the quantity and the budget for the data.
+From this, the price per data can be calculated as:
+
+```
+price_per_data = deposit / max_num_data
+```
```

Comment:
I think that it will be good for explaining how does the remainder calculated.

Reply:
Actually, we don't calculate remaining budget.
When consumer deactivate the deal, just balance goes back to consumer (by using `bankkeeper.GetBalance`).